grafana/dashboards/charge-level.json

```diff
@@ -167,8 +167,8 @@
           "format": "percent",
           "label": "Charge Level",
           "logBase": 1,
-          "max": null,
-          "min": null,
+          "max": 100,
```

Comment:
I believe this should be "100" (with the quotes) - when editing in Grafana directly it also makes quotes around the numbers...

Reply:
Thanks, fixed.